Utils/HandlerUtil.py

```diff
@@ -73,6 +73,15 @@ class HandlerContext:
     def __init__(self,name):
         self._name = name
         self._version = '0.0'
+        self._config_dir = None
+        self._log_dir = None
+        self._log_file = None
+        self._status_dir = None
+        self._heartbeat_file = None
+        self._seq_no = -1
+        self._status_file = None
+        self._settings_file = None
+        self._config = None
```

Comment:
PR looks good to me. Thanks @jasonzio for your quick review.

@anandramakrishna , just wondering if these newly added members will be used later?